src/marvin/components/library/ai_functions.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
```

Comment:
Something about having the component library nested under the components themselves is odd to me - like the library should be higher-level than components, not a subset of them. Maybe `marvin.library` or `marvin.cookbook`?

Reply:
that makes sense - i first had it as `marvin.library` and got weirded out that `marvin.prompts.library` was different